doc/content/gateways/lora-basics-station/_index.md

```diff
@@ -0,0 +1,9 @@
+---
+title: "LoRa Basics™ Station"
```

Comment:
Isn't it LoRa(TM) @laurensslats?

Reply:
https://github.com/lorabasics/basicstation